build.gradle

```diff
@@ -72,6 +72,7 @@ testing {
                 implementation "org.junit.jupiter:junit-jupiter:$junit_version"
                 implementation "org.mockito:mockito-core:5.+"
                 runtimeOnly "org.junit.platform:junit-platform-launcher"
+                implementation "org.wiremock:wiremock:3.5.2"
```

Comment:
```suggestion
                testImplementation "org.wiremock:wiremock:3.5.2"
```

Reply:
I tried to switch that out and it errors with the below, I can't find any docs but because we're in the `dependencies` of the `testing`/`test` block are they implicitly `testImplementation`s?

```
Could not find method testImplementation() for arguments [org.wiremock:wiremock:3.5.2] on object of type org.gradle.api.plugins.jvm.internal.DefaultJvmComponentDependencies.
```